tests/integration/requirements.txt

```diff
@@ -1,5 +1,8 @@
 juju
 pytest
+Pillow
 requests
 pytest-cov
-pytest-operator
\ No newline at end of file
+pytest-operator
+python-swiftclient
+python-keystoneclient
```

Comment:
Is this needed? I couldn't see it get imported anywhere?

Reply:
`python-keystoneclient` is required for `python-swiftclient` to use Identity API v3.